pkg/controller/job/job_controller.go

```diff
@@ -447,6 +447,16 @@ func (jm *Controller) deletePod(logger klog.Logger, obj interface{}, final bool)
 	jm.enqueueSyncJobBatched(logger, job)
 }
 
+func (jm *Controller) addJob(logger klog.Logger, obj interface{}) {
+	jm.enqueueSyncJobImmediately(logger, obj)
+	jobObj, ok := obj.(*batch.Job)
+	if ok {
+		if controllerName, uses := managedByExternalController(jobObj); uses {
+			metrics.JobByExternalControllerTotal.WithLabelValues(controllerName).Inc()
```

Comment:
any reason why you didn't add a log here?
could be overkill..

Reply:
Yeah, I think this would be over-kill, this is just incrementing the metric. 

In similar situations we don't log in the recording functions: [recordJobPodsCreationTotal](https://github.com/kubernetes/kubernetes/blob/50bf3a2060ea798600af1a4c125e0b62d08e9680/pkg/controller/job/job_controller.go#L1920-L1938), or [recordJobPodFinished](https://github.com/kubernetes/kubernetes/blob/50bf3a2060ea798600af1a4c125e0b62d08e9680/pkg/controller/job/job_controller.go#L1831-L1860)